keyboards/keaboard/rev1/config.h

```diff
@@ -0,0 +1,18 @@
+/*
```

Comment:
This file can be deleted.

Reply:
Removing this file breaks my builds because `config.h` is included elsewhere in QMK. There was a PR from @waffle87 that removes those includes but it is not yet in master for me to use.

https://github.com/qmk/qmk_firmware/pull/19816#discussion_r1103724967

Reply:
The CI build is failing after removing this file.

Reply:
Ah, I thought the CI merged the branches but I guess not. In any case, it will definitely work once merged.